<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferEntryDialog</class>
 <widget class="QDialog" name="TransferEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transfer Entry Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>95</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Create Entry</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Clear Fields</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>58</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Defendant Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="defendant_name"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Case No.:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="case_no"/>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Assigned Date:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="assigned_date">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Assigned Judge:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="assigned_judge">
      <item>
       <property name="text">
        <string>Judge Marianne Hemmeter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Judge Kyle Rohrer</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Transferred to:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="transferred_judge">
      <item>
       <property name="text">
        <string>Judge Marianne Hemmeter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Judge Kyle Rohrer</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="1">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Defendant Address:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Defendant City:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Defendant State:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Defendant Zipcode:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="defendant_address"/>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="defendant_city"/>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="defendant_zipcode"/>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="defendant_state">
      <item>
       <property name="text">
        <string>OH</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IN</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>defendant_name</tabstop>
  <tabstop>case_no</tabstop>
  <tabstop>assigned_date</tabstop>
  <tabstop>assigned_judge</tabstop>
  <tabstop>transferred_judge</tabstop>
  <tabstop>defendant_address</tabstop>
  <tabstop>defendant_city</tabstop>
  <tabstop>defendant_state</tabstop>
  <tabstop>defendant_zipcode</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TransferEntryDialog</receiver>
   <slot>createEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>defendant_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>case_no</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>TransferEntryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>defendant_address</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>defendant_city</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>defendant_zipcode</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createEntry()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
